books/free-programming-books-ar.md

```diff
@@ -10,21 +10,23 @@
 * [Introduction to Programming in Arabic](#introduction-to-programming-in-arabic)
 * [Java](#java)
 * [JavaScript](#javascript)
-    * [Vue.js](#vuejs)
```

Comment:
Alphabetize the listing 🤔 says the linter 🤓

Sorting resources and spaces are the most common issues in this repository 😉

Reply:
* [Arduino](#arduino)
* [Artificial Intelligence](#artificial-intelligence)
* [C#&lrm;](#csharp)
* [DB & DBMS](#db--dbms)
* [HTML and CSS](#html-and-css)
* [Introduction to Programming in Arabic](#introduction-to-programming-in-arabic)
* [Java](#java)
* [JavaScript](#javascript)
* [Vue.js](#vuejs)
* [Linux](#linux)
* [Open Source Software](#open-source-software)
* [Operating System](#operating-systems)
* [PHP](#php)
* [Python](#python)
* [Django](#django)
* [Raspberry Pi](#raspberry-pi)
* [Ruby](#ruby)
* [Scratch](#scratch)
* [Security](#security)
* [SEO](#seo)
* [SQL](#sql)
* [PostgreSQL](#postgresql)
* [UI/UX](#uiux)
* [DataScience](#data-science)

⬇️ Suggested change

* [Arduino](#arduino)
* [Artificial Intelligence](#artificial-intelligence)
* [C#&lrm;](#csharp)
* [DataScience](#data-science)
* [DB & DBMS](#db--dbms)
* [Django](#django)
* [HTML and CSS](#html-and-css)
* [Introduction to Programming in Arabic](#introduction-to-programming-in-arabic)
* [Java](#java)
* [JavaScript](#javascript)
* [Linux](#linux)
* [Open Source Software](#open-source-software)
* [Operating System](#operating-systems)
* [PHP](#php)
* [PostgreSQL](#postgresql)
* [Python](#python)
* [Raspberry Pi](#raspberry-pi)
* [Ruby](#ruby)
* [Scratch](#scratch)
* [Security](#security)
* [SEO](#seo)
* [SQL](#sql)
* [UI/UX](#uiux)
* [Vue.js](#vuejs)